README.md

```diff
@@ -7,8 +7,12 @@
   <br />
   <a title="Learn more about Separators" href="http://github.com/alefragnani/vscode-separators"><img src="https://raw.githubusercontent.com/alefragnani/vscode-separators/master/images/vscode-separators-logo-readme.png" alt="Separators Logo" width="70%" /></a>
 </p>
+## Seperators
```

Comment:
I prefer the original README structure, with **What’s New**  and **Support** sections on top. The extension description goes below them.